### PR TITLE
Use tendermint/tm-db

### PR DESCRIPTION
**Pull request by tnasu**
## Description

* Use `tendermint/tm-db-v0.6.6`: https://github.com/tendermint/tendermint/blob/v0.34.19/go.mod#L37
  * Stop using `line/tm-db`

## Background
* `line/lbm-sdk` stop using `line/tm-db` and etc. 
  * https://github.com/line/lbm-sdk/pull/605
* `line/tm-db` is not maintained quickly and effectively
* `line/ostracon` back-port from the upstream of `tendermint/tendermint`, then it makes a lot of conflicts while using `linie/tm-db`

I think using `tendermint/tm-db` has many merites.

### CODE REVIEW

File: node/node_test.go [hunk 1]
```diff
@@ -144,7 +143,7 @@ func TestNodeSetAppVersion(t *testing.T) {
 	require.NoError(t, err)
 
 	// default config uses the kvstore app
-	var appVersion uint64 = kvstore.ProtocolVersion
+	var appVersion = kvstore.ProtocolVersion
```
Comment (tnasu) [inline node/node_test.go:146]: Changed by my ide's save action...

File: rpc/core/blocks_test.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package core
 
 import (
-	errors "errors"
+	"errors"
```
Comment (tnasu) [inline rpc/core/blocks_test.go:4]: Changed by my ide's save action...

File: config/config.go [hunk 3]
```diff
@@ -231,14 +230,6 @@ type BaseConfig struct { //nolint: maligned
 
 // DefaultBaseConfig returns a default base configuration for an Ostracon node
 func DefaultBaseConfig() BaseConfig {
-	dbs := metadb.AvailableDBBackends()
```
Comment (tnasu) [inline config/config.go:234]: `AvailableDBBackends` is `line/tm-db` original.
* https://github.com/line/tm-db/pull/15

**Comment by torao** [comment]
The explanation seems too brief.
Is there an Issue or related PR that explains why this change is necessary?

**Comment by tnasu** [comment]
@torao Updated the description. See that. Thanks.

**Review by torao — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]
LGTM

**Pull request merged**